app/Http/Controllers/IndexController.php

```diff
@@ -14,12 +14,12 @@
 
 class IndexController extends Controller
 {
-    public function index(){
+    public function index() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->